package.json

```diff
@@ -20,8 +20,10 @@
     "docs:serve": "mkdocs serve",
     "link:all": "lerna exec yarn link",
     "lint": "eslint --no-eslintrc --c ./.eslintrc.js --cache --report-unused-disable-directives --format codeframe --ext js,jsx \".*.js\" packages",
-    "release": "lerna publish",
-    "release:preview": "lerna publish --skip-git --skip-npm",
+    "release:prepare": "lerna version",
+    "release:publish": "lerna publish from-package",
+    "release:publish-next": "yarn release:publish --dist-tag next",
+    "release:ci": "lerna publish prerelease --registry http://localhost:4873 --no-git-reset --yes",
```

Comment:
I don't see `prerelease` as a valid positional for the `lerna publish` command, am I missing something?

https://github.com/lerna/lerna/tree/master/commands/publish#positionals

Reply:
It also appears that `SKIP_CHANGELOG` isn't being respected:

https://travis-ci.org/neutrinojs/neutrino/jobs/506989820#L606

Reply:
I think updating the `version` script to the following should work:

```sh
[ -z $SKIP_CHANGELOG ] || yarn changelog --package && git add CHANGELOG.md
```

Reply:
> I don't see `prerelease` as a valid positional for the `lerna publish` command, am I missing something?

The `lerna publish` command is a superset of `lerna version`, so supports the options version supports:
https://github.com/lerna/lerna/tree/master/commands/version#semver-bump

I'd originally wanted to have the `yarn release:ci` command follow the actual release process as closely as possible - ie: make it do something like:
`yarn release:prepare prerelease --yes && yarn release:publish --registry http://localhost:4873 --yes`

However it turns out when `lerna publish` is run using either of the `from-git` or `from-package` options, lerna insists that the working directory is clean, which it won't be in CI, since we deliberately miss out the ```Commit the changes using: `git commit -a -m 'vN.N.N'``` step that is performed during a real release.

As such, I avoided this by making `yarn release:ci` perform the publish in one step, by running `lerna publish` in "do all steps" mode (ie: by specifying a version bump, rather than `from-git` or `from-package`).

> It also appears that `SKIP_CHANGELOG` isn't being respected:

Yeah I saw that locally on Windows too. Yarn must not propagate the modified env for sub-commands, strange.